wyvern/cli/commands.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+import getpass
+import importlib
+import os
+import shutil
+import zipfile
+from pathlib import Path
+from typing import Optional
+
+import requests
+import typer
+import uvicorn
+from typing_extensions import Annotated
+
+app = typer.Typer()
+WYVERN_TEMPLATE_URL = "https://codeload.github.com/Wyvern-AI/wyvern-starter/zip/main"
+
+
+def _replace_info(project: str, author: Optional[str] = None):
+    toml_file_path = os.path.join(project, "pyproject.toml")
+    if os.path.isfile(toml_file_path):
+        author = author or getpass.getuser() or ""
+        with open(toml_file_path, "r") as file:
+            file_content = file.read()
+        file_content = file_content.replace("wyvern-starter", project)
```

Comment:
what happens if the passed `project: str` is a path that contains `/`?

In arg help, we define project as `Name and path to initialize the project`

Reply:
let's just do name!

i'll add a name validation so that the project cannot have a `/` in it.